install/empty_data.php

```diff
@@ -1733,14 +1970,37 @@
 $ADDTODISPLAYPREF['NetworkPort'] = [3, 30, 31, 32, 33, 34, 35, 36, 38, 39, 40];
 $ADDTODISPLAYPREF['USBVendor'] = [10, 11];
 $ADDTODISPLAYPREF['PCIVendor'] = [10, 11];
-
 foreach ($ADDTODISPLAYPREF as $type => $options) {
    $rank = 1;
    foreach ($options as $newval) {
       $tables['glpi_displaypreferences'][] = [
          'itemtype'  => $type,
          'num'       => $newval,
          'rank'      => $rank++,
+         'is_main'   => 1
+      ];
+   }
+}
+
+/** add display preferences for sub items */
+$ADDTOSUBDISPLAYPREF = [];
+$ADDTOSUBDISPLAYPREF['Contract'] = [3, 4, 29, 5];
```

Comment:
"Initial contract period" is missing.
```suggestion
$ADDTOSUBDISPLAYPREF['Contract'] = [3, 4, 29, 5, 8];
```